### DISCUSSION TITLE
PMGH is dead, but this isn't the end for Prodigy Hacks

### DISCUSSION THREAD

**Discussion by afkvido** [ProdigyErrorCodes/ProdigyErrorCodes#25] [Announcements]
# PMGH is dead, but this isn't the end for Prodigy Hacks.

First of all, read the below message from BoredFish.

### `BoredFish#4269`, `#hack-announcements`, `discord.gg/XQDfbfq`, `03/26/2022`

```markdown
Important message for @everyone:

First we'd like to thank everyone for being with us. For the past 2.5 years, we've maintained hacks for the math game Prodigy.

Our original goal, which was raising awareness about the security of Prodigy has (mostly) been accomplished!
While we would love to continue maintaining hacks, we collectively have had less and less time to focus on development.
Server costs are also surprisingly expensive..! Who knew?

For these reasons, As of **April 10th, 2022**, we will shut down our instance of P-NP.
**Hacks for the most part will no longer work.**

We will keep our source code on GitHub, where anyone can continue development by forking our repositories.
We'll continue to review pull requests, but patched hacks most likely will remain patched.

If you would like to run your own instance of the hacks, we'll soon release instructions on how to run P-NP.
This discord server will move from a Prodigy Hacking server to a Coding focused server over the next month.

Thank you all for understanding, and being with us for the past few years.
So long, and thanks for all the fish. 
```

That's pretty sad, but there's not too much problem since they'll keep the repo... right?

It gets a lot, a lot fucking worse.

### `BoredFish#4269`, `#hack-announcements`, `discord.gg/XQDfbfq`, `04/06/2022`

```markdown
@everyone Repository is now archived. This means it is read only. Since prodigy has DMCAed us, we cannot reproduce
hacks without legal problems. So, it is now actually game over, we cannot release P-NP instructions OR the code
anymore. This is really the end, no joke. Not the end for the server though, and us. The only thing doing away
is the hacks. So long, and thanks for all the fish again.
```

THEN THE REPOS GET DELETED **WITHOUT WARNING**.

## bless the demon king
i had made local clones of the repos before they were deleted.

# [ProdigyPNP](https://github.com/ProdigyPNP)
I've made an organization that's dedicated to getting digital history of PMGH, and **reviving the hacks.**
I'll make a ProdigyPNP forum soon, for now we can chat here

**Comment by donaldli2020** [comment]
I'm Canadian. There's nothing called DMCA.

**Reply by donaldli2020** [reply]
lmao

**Reply by afkvido** [reply]
the DMCA in the USA is bad as hell

**Reply by donaldli2020** [reply]
Prodigy is Canadian. Hey Prodigy! Have fun on your vacation to the USA!
Source: https://www.google.com/search?q=prodigy+headquarters

**Reply by donaldli2020** [reply]
Maps https://www.google.com/maps/dir/Oakville,+Ontario/Buffalo,+NY,+USA/@43.1886835,-79.6505322,10z/data=!4m14!4m13!1m5!1m1!1s0x882b5b8b8f729979:0x5037b28c7231bd0!2m2!1d-79.6876659!2d43.467517!1m5!1m1!1s0x89d3126152dfe5a1:0x982304a5181f8171!2m2!1d-78.8783689!2d42.8864468!3e0

**Reply by afkvido** [reply]
BRUH. i just realized

https://www.google.com/maps/place/Prodigy+Education/data=!3m1!4b1!4m5!3m4!1s0x882b61b1b1a7c0fd:0x679d26f9f2ad7fdb!8m2!3d43.4462151!4d-79.6940185
